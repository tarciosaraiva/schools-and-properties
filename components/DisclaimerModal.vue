<template>
  <div>
    <input id="modal-control-disclaimer" type="checkbox" class="modal" />
    <div role="dialog" aria-labelledby="dialog-title">
      <div class="card large">
        <label for="modal-control-disclaimer" class="modal-close" ></label>
        <h3 id="dialog-title" class="section">About</h3>
        <p>The information provided by this website is a collation of 3 major data sources, listed in the attributions modal accessed by the link at the footer of this website.</p>
        <p>The schools displayed in the map are only the ones who have a matching of 90 or above in <a href="https://www.bettereducation.com.au/" target="blank">Better Education</a> website so it's likely you will notice some schools may be missing from the list.</p>
      </div>
    </div>
  </div>
</template>

<script lang="ts">
import Vue from 'vue'
export default Vue.extend({})
</script>
