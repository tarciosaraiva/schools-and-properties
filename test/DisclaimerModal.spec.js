import { mount } from '@vue/test-utils'
import DisclaimerModal from '@/components/DisclaimerModal.vue'

describe('DisclaimerModal', () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(DisclaimerModal)
  })

  test('is a Vue instance', () => {
    expect(wrapper.vm).toBeTruthy()
  })

  test('renders correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
