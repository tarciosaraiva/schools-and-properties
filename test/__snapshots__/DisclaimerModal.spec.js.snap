// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DisclaimerModal renders correctly 1`] = `
<div>
  <input class="modal" id="modal-control-disclaimer" type="checkbox">
  <div aria-labelledby="dialog-title" role="dialog">
    <div class="card large">
      <label class="modal-close" for="modal-control-disclaimer"></label>
      <h3 class="section" id="dialog-title">Disclaimer</h3>
      <p>The information provided by this website is a collation of 3 major data sources, listed in the attributions modal accessed by the link at the footer of this website.</p>
      <p>The schools displayed in the map are only the ones who have a matching of 90 or above in <a href="https://www.bettereducation.com.au/" target="blank">Better Education</a> website so it's likely you will notice some schools may be missing from the list.</p>
    </div>
  </div>
</div>
`;
